### PR TITLE
[ISSUE-713] Add new target for k8s compatibility testing

### LINKED ISSUE #713: Add test-sanity-ci target for k8s compatibility testing
**Is your feature request related to a problem? Please describe.**
Need to test CSI on as many as possible k8s versions, and at the same time need to spend as little as possible time for compatibility testing.

**Describe the solution you'd like**
Proposed solution is to run as compatibility test only 1 test from e2e suite.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by misanthropicat**
### Resolves #713 

Added test-sanity-ci target
Ginkgo focus is parameterized by job parameter

## PR checklist
- [x] Add link to the issue
- [x] Choose Project
- [x] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

## Testing
Custom-ci is passed on: v1.19.11, v1.20.7, v1.21.2, v1.22.0
Custom-ci is failed on v1.23.0
Links are in corresponding Jira.

### CODE REVIEW

File: Makefile.validation [hunk 1]
```diff
@@ -37,6 +37,9 @@ test-ci:
 test-short-ci:
 	${GO_ENV_VARS} CI=true CSI_VERSION=${CSI_VERSION} OPERATOR_VERSION=${OPERATOR_VERSION} go test -v test/e2e/baremetal_e2e_test.go -ginkgo.v -ginkgo.progress -ginkgo.failFast -timeout-short-ci=${SHORT_CI_TIMEOUT} -kubeconfig=${HOME}/.kube/config -chartsDir ${CHARTS_DIR} -timeout 0 > log.txt
 
+test-sanity-ci:
+	${GO_ENV_VARS} CI=true CSI_VERSION=${CSI_VERSION} OPERATOR_VERSION=${OPERATOR_VERSION} go test -v test/e2e/baremetal_e2e_test.go -ginkgo.focus="\[Testpattern: Dynamic PV \(default fs\)\] volumes should store data" -ginkgo.v -ginkgo.progress -all-tests  -kubeconfig=${HOME}/.kube/config -chartsDir ${CHARTS_DIR} > log.txt
```
Comment (rkiselenko) [inline Makefile.validation:41]: Can we have focus as parameter too ?

**Review by rkiselenko — CHANGES_REQUESTED** [review]

File: Makefile.validation (hunk 1 above)
Comment (misanthropicat) [reply Makefile.validation:41]: Okay, will do.

**Review by mishoyama — APPROVED** [review]

File: Makefile.validation (hunk 1 above)
Comment (rkiselenko) [reply Makefile.validation]: The default version of this variable must exists.

**Review by rkiselenko — APPROVED** [review]

File: Makefile.validation [hunk 2]
```diff
@@ -37,6 +37,9 @@ test-ci:
 test-short-ci:
 	${GO_ENV_VARS} CI=true CSI_VERSION=${CSI_VERSION} OPERATOR_VERSION=${OPERATOR_VERSION} go test -v test/e2e/baremetal_e2e_test.go -ginkgo.v -ginkgo.progress -ginkgo.failFast -timeout-short-ci=${SHORT_CI_TIMEOUT} -kubeconfig=${HOME}/.kube/config -chartsDir ${CHARTS_DIR} -timeout 0 > log.txt
 
+test-sanity-ci:
+	${GO_ENV_VARS} CI=true CSI_VERSION=${CSI_VERSION} OPERATOR_VERSION=${OPERATOR_VERSION} go test -v test/e2e/baremetal_e2e_test.go -ginkgo.focus=${SANITY_TEST} -ginkgo.v -ginkgo.progress -all-tests  -kubeconfig=${HOME}/.kube/config -chartsDir ${CHARTS_DIR} > log.txt
```
Comment (safronovD) [inline Makefile.validation:41]: What if default for `SANITY_TEST`?

**Review by safronovD — APPROVED** [review]

File: Makefile.validation (hunk 2 above)
Comment (misanthropicat) [reply Makefile.validation:41]: `\[Testpattern: Dynamic PV \(default fs\)\] volumes should store data`
It'll be available as a default parameter value in custom-ci pipeline.

**Pull request merged**